docs/pages/develop/development-builds/development-workflows.mdx

```diff
@@ -139,40 +139,3 @@ You can now view and load EAS Updates in your development build via the `Extensi
 When you create a development build of your project, you'll get a stable environment to load any changes to your app that are defined in JavaScript or other asset-related changes. Other changes to your app, whether defined directly in **android** and **ios** directories or by packages or SDKs you choose to install, will require you to create a new build of your development build.
 
 To enforce an API contract between the JavaScript and native layers of your app, you should set the [`runtimeVersion`](/distribution/runtime-versions) value in the app config. Each build you make will have this value embedded and will only load bundles with the same `runtimeVersion`, in both development and production.
-
-## Build locally with Android Studio and Xcode
```

Comment:
should we link out to the relevant docs for building here?

Reply:
I was thinking of adding a link for Local development build as next steps on this page: https://docs.expo.dev/develop/development-builds/introduction/

and then remove Installation page (https://docs.expo.dev/develop/development-builds/installation/) and include any relevant info from this installation page in the Create development build (https://docs.expo.dev/develop/development-builds/create-a-build/). The reason behind this you raised a great appoint about installation guide as a recommended step sometime back.

(_Sidenote:_ I'm assuming I can install `expo-dev-client` and still use local development build and remove native directories and call it a managed workflow but correct if my assumption is wrong.)

Reply:
Also, this section was previously mentioned in Tools, Workflows and Extensions which doesn't make sense to bury the local development info in the last of this section (I don't think developers will be able to find if add the link here quite easily).